### PR TITLE
Environment Variable to override DB connection string

### PR DESCRIPTION
**Pull request by mrhut10**
Environment Variable to override DB connection string

this enables someone copy and pasting a connection string almost straight out of a hosted DB like Mongo Atlas service other similar alternatives, if anyone is like me i found it difficult converting from there provided string back to the current env variables cezerin requires.

My testing went as far as confirming locally a new project connected and authenticated to MongoDB Atlas database, however hoping someone can confirm works on current project

### CODE REVIEW

File: config/server.js [hunk 1]
```diff
@@ -1,13 +1,12 @@
 // config used by server side only
-
+const dbConectString = process.env.DB_STRING || null;
```
Comment (mrhut10) [inline config/server.js]: setting DB_STRING environmental variable will override any other DB settings variable.

this helped me greatly getting a new project working with the hosted mongoDB atlas service

**Comment by mrhut10** [comment]
after looking I have a feeling this might be a similar fix to pr#434

does everyone feel this is too similar to pr#434

**Comment by mrhut10** [comment]
I've updated this so no merge conflicts @restmount although this fix might be similar to #434 would be worth looking at both of them before deciding which one to merge

**Comment by sonufrienko** [comment]
@mrhut10 
Thanks.
It's fixed in #434

**Pull request closed without merging**